config/webpack/loaders/devSiteCodeblock.js

```diff
@@ -0,0 +1,37 @@
+const path = require('path');
+const findSource = require('../loaderUtils/findSource');
+
+/**
+ * Load the contents of the file into the codeblock
+ * This loader expects to be chained with the mdx loader.
+ */
+const loader = async function loader() {
+  const callback = this.async();
+
+  const exampleSource = this.resourcePath;
+  const { source } = findSource(exampleSource);
+
+  this.resolve('', source, (err, result) => {
```

Comment:
what is result in this case?

Reply:
The exact file path of the resoved source file. The source file does not have the file extension. I use resolve to ensure the file exists.

Reply:
ah is `this.resovle` equivalent to `path.resovle` in this case? I was thinking it was in reference to the async promise resolve

Reply:
https://webpack.js.org/api/loaders/#thisresolve